src/lib.rs

```diff
@@ -236,6 +236,11 @@ impl From<String> for EarlyExit {
 /// was unsuccessful or if information like `--help` was requested.
 pub fn from_env<T: TopLevelCommand>() -> T {
     let strings: Vec<String> = std::env::args().collect();
+    let cmd = std::path::Path::new(&strings[0])
```

Comment:
this doesn't seem to be used?

Reply:
Oops. I made a new branch with the changes.  Missed that.

Reply:
Ok, being used now.